README.md

```diff
@@ -1,191 +1,264 @@
 # @eik/node-client
 
-The Eik Node.js client facilitates loading Eik config and providing URLs to assets on an Eik server or in local development plus loading import maps from the Eik server.
+This is an Eik utility for servers running on Node. With it you can:
+
+- generate different URLs to assets on an Eik server depending on environment (development vs production).
+- get the import maps you have configured in `eik.json` from the Eik server, should you want to use them in the HTML response.
 
 ## Install
 
 ```sh
 npm install @eik/node-client
 ```
 
-## Basic usage
-
-```js
-import EikNodeClient from '@eik/node-client';
-
-const client = new EikNodeClient({
-    development: false,
-    base: '/public'
-});
-
-await client.load();
-
-const scriptPath = client.file('/a-script-file.js');
-```
-
-## Description
-
-This module will load Eik config from either an `eik.json` file or from values set in `package.json` and then use these values to provide absolute URLs to assets on a Eik server. In addition to this it's possible to set a `base` URL which will be used as the "base root" for files when this module is set in development mode. This makes it easy to retrieve absolute URLs to assets on a Eik server when an application is running in production but also get URLs to the same assets when in development.
+## Usage
 
-In addition this module can also download the import maps defined in Eik config and provide these for inclusion in an application.
+The most common use case for this module is linking to a file.
 
-The following will use the information in Eik config and provide an absolute URL to a file on an Eik server:
+When developing you typically want to use a local version of the file, then link to the published version on Eik when running in production.
```

Comment:
This key difference between development and production I think could be emphasised a bit more, as having the client in development mode means it does something entirely different than in production 🤔 

I see it mentioned in the method docs, which is really good, I was just thinking for my self I would read this passage and just glance over the method docs. Maybe we could write something about this difference in behaviour here too.